database-plugins/src/main/java/co/cask/hydrator/plugin/DBUtils.java

```diff
@@ -224,7 +227,6 @@ public static DriverCleanup ensureJDBCDriverIsAvailable(Class<? extends Driver>
       case Types.JAVA_OBJECT:
       case Types.OTHER:
       case Types.REF:
-      case Types.ROWID:
       case Types.SQLXML:
       case Types.STRUCT:
         throw new SQLException(new UnsupportedTypeException("Unsupported SQL Type: " + sqlType));
```

Comment:
is there any way to change this exception message to include the name of the sqlType instead of the number?

Reply:
I am not sure how we can do it since we have not inferred the type.